packages/fev4-fpm-writer/templates/column/ext/CustomColumnFragment.xml

```diff
@@ -0,0 +1,10 @@
+<core:FragmentDefinition xmlns:core="sap.ui.core" xmlns="sap.m"
+	xmlns:l="sap.ui.layout">
+	<l:VerticalLayout <%- locals.eventHandler ? ' core:require="{ handler: \'' + eventHandler.fileName + '\'}"' : '' %>>
```

Comment:
Adding a vertical Layout as the basic sample we have for a column is really not nice, i'm not even sure they should be used.

Why don't you just add the control directly ?

Reply:
That's what we do in V2, an that's what is documented for V4: https://ui5.sap.com/#/topic/d525522c1bf54672ae4e02d66b38e60c.html

Reply:
I'll double check with the colleagues and potentially ask for an update in docu for this

Reply:
@nlunets as discussed,  the `VerticalLayout` has been removed.